Release/v2.0.0 (#43)
containeroo/helminator: README.md

## Introduction

Helminator scans your Ansible playbook for helm and helm_repository tasks.

> It then checks if there is an update to any of the defined Helm charts available. If configured, it creates a branch and merge request and can also sends out a Slack notification.

Reviewer: ```suggestion
It then checks if there is an update to any of the defined Helm charts available. If configured, it creates a branch and merge request and/or send out a Slack notification.
```
